### PR TITLE
[DONT MERGE YET] Problem: csv import gets broken by rackcontroller-0

### PR DESCRIPTION
**Pull request by EldoreiJK**
Solution: Added support for rackcontroller-0 to CSV import.

### CODE REVIEW

**Comment by EldoreiJK** [comment]
Still need some tests, and I'm requesting your comments.

File: src/db/dbhelpers.h [hunk 1]
```diff
@@ -53,59 +53,59 @@ typedef std::function<void(const tntdb::Row&)> row_cb_f ;
 
 template <typename T>
 struct db_reply{
-    int status; // ok/fail
```
Comment (jimklimov) [inline src/db/dbhelpers.h]: It seems that the changes in `dbhelpers.h` are all about breaking indentation, and nothing functional. Can you pluck this one out of the PR please?

**Comment by jimklimov** [comment]
Probably `libmlm-dev` needs to be uncommented in `.travis.yml`.

**Comment by EldoreiJK** [comment]
Good point about useless changes, reverted both 2 files with no more changes than whitespaces.

**Pull request merged**